docs/Theory_cluster_batch_scheduling.rst

```diff
@@ -1,15 +1,132 @@
 The workload manager
 ********************
 
-Once connected to a login node, presumably with SSH, you can issue a
-job execution request to what is called the workload manager. The
-workload manager used at Mila and Compute Canada clusters is called
-:ref:`Slurm <slurmpage>`.  The workload manager's main role is to find a
-place to run your program in what is simply called a *job*. This
-"place" is in fact one of many computers synchronised to the manager
-which are called :ref:`The compute nodes`.
-
-In fact it's a bit trickier than that, but we'll stay at this abstraction level
-for now.
-
-.. include:: Theory_cluster_slurm.rst
+On a cluster, users don't have direct access to the compute nodes but
+instead connect to a login node and add jobs to the workload manager
+queue. Whenever there are resources available to execute these jobs
+they will be allocated to a compute node and run, which can be
+immediatly or wait for several days.
```

Comment:
```suggestion
immediately or after a wait of several days.
```

Reply:
It is clear to me but can be ambiguous. It's from immediately to several day. Can be in between (e.g. a couple of hours).

Reply:
I went with "after a wait of up to several days"